episodes/06-ignore.md

```diff
@@ -17,14 +17,39 @@ exercises: 0
 
 ::::::::::::::::::::::::::::::::::::::::::::::::::
 
+## What not to put under Version Control?
+
+In the section called “What Not to Put Under Version Control” from [G. Wilson et al. 2017](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1005510#sec014) on Good Enough Practices in Scientific Computing, authors emphasize, among other notes, in three points: 
+
+- Version control systems are __optimized for plain-text files__; 
+- __Avoid raw data__, since it should not change;
+- __Avoid intermediate files or results__ that can be *re-generated* from raw data and software.
+
+::::::::::::::::: checklist
+
+### Good Practice
```

Comment:
This box is repeating information in the bullet points above. These points could be merged.

Reply:
I agree. I decided to relocate it at the end of the episode content and as a reminder before the challenges in https://github.com/epiverse-trace/git-rstudio-basics/pull/13/commits/0b4de95bb7c6ac00ccd8e886ca80b32b1727b2f8